_data/simple-icons.json

```diff
@@ -6007,9 +5999,7 @@
             "hex": "FF160B",
             "source": "https://en.wikipedia.org/wiki/File:NJPW_World_Logo.svg",
             "aliases": {
-                "aka": [
-                    "NJPW"
-                ],
+                "aka": ["NJPW"],
```

Comment:
Could you revert all these unrelated changes for our sanity? I'm guessing your editor autoformatted this. I understand this can be a bit annoying for you to do but from our side different contributors will end up formatting the code different each time. The problem with this is that it renders the commit history _useless_ as the commit that last changed this isn't actually related to the value. I hope you understand.

If you want, you could consider making a contribution that adds a formatter that enforces the style we currently use.

Reply:
I don't see that line in my file.. maybe it is a mismatch?

Reply:
This line must be in your local file unless you have uncommitted changes. If you need help reverting these changes feel free to ask